src/js/redux/actions/search/searchFilterActions.js

```diff
@@ -31,3 +31,14 @@ export const updateGenericFilter = (state) => ({
 export const clearAllFilters = () => ({
     type: 'CLEAR_SEARCH_FILTER_ALL'
 });
+
+export const updateSelectedLocations = (state) => ({
+    type: 'UPDATE_SELECTED_LOCATIONS',
+    location: state.location,
+    direction: state.direction
+});
+
+export const setAutocompleteLocations = (state) => ({
```

Comment:
Is it consistent to put the autocomplete action in this file when it is generally handled in separate autocomplete files elsewhere in the Redux implementation?